packages/frontapp/src/app/components/Modal/index.stories.tsx

```diff
@@ -0,0 +1,23 @@
+import Modal, { IPropsModal } from './index';
+import { Story } from '@storybook/react';
+import { useState } from 'react';
+
+export default {
+  title: 'Modal',
+  component: Modal,
+};
+
+const Wrapper = (props: any) => {
+  const [active, setActive] = useState(true);
+  return <Modal active={active} setActive={setActive} {...props} />;
+};
```

Comment:
I can't see any content on Storybook when opening/closing a modal. Can you add some `children` to Modal e.g. a black square / text content ?